OMNA/LwM2M/LwM2MObjects.html

```diff
@@ -165,10 +161,10 @@ <h3 id="omalabel"><strong>oma-label</strong> Objects Produced by OMA</h3>
         <tr>
             <td>urn:oma:lwm2m:oma:8</td>
             <td>
-                <a title="download xml file" href="">8</a>
+                <a>8</a>

```

Comment:
An <a> tag has no semantic meaning without an href. If it's not a link that goes anywhere, then either remove the link totally or put in href = "" . Preferably the former, as why have a link at all if it doesn't go anywhere?